ftw/simplelayout/browser/resources/main.js

```diff
@@ -70,7 +70,7 @@
 
     var target = $("body");
 
-    var baseUrl = $("body").data("base-url") ? $("body").data("base-url") + "/" : $("base").attr("href");
+    var baseUrl = $("body").data("base-url") && $("base").length === 0 ? $("body").data("base-url") + "/" : $("base").attr("href");
```

Comment:
@maethu If I understand that correctly you are reverting the priority here. I think you could revert the condition and it would be easier understandable:
from: a && !b ? a : b
to:  b ?  b : a
Isn't that the same / does what you want?

Reply:
yep ;-) and done